README.md

```diff
@@ -39,6 +40,100 @@ Note that politeiawww does not provide HTML output.  It strictly handles the
 JSON REST RPC commands only.  The GUI for politeiawww can be found at:
 https://github.com/decred/politeiagui
 
+## Development 
+
+#### 1. If you haven't install Go, check it out the [docs fot it](https://golang.org/doc/install)
+#### 2. Get this repository properly cloned and installed. 
+#### 3. If you have properly setted your enviroment you will be able to run Politeia's components by simple typing its name (e.g: `politeiawwww`) on your terminal (p.s It won't fully work untill you set its configuration).
+#### 4. Finding your configuration files path: 
+* To make things simpler during development is handy having a configuration file for your components. Specially `politeiad` and `politeiawww` so you don't have to pass the parameters every time you run it. 
+* You may find the configuration files under the path:  
+  - **Mac OS** `/Users/<username>/Library/Application Support/Politeiad/politeiad.conf`
+  - **Windows** `C:\Users\<username>\AppData\Local\Politeiad/politeiad.conf`
+  - **Ubuntu** `~/.politeiad/politeiad.conf`
+
+#### 5. Set the configuration files:
+  
+Add the following configurations:
```

Comment:
You can probably leave this section for listing good defaults but you should tell the developer to copy and change the [`sample-politeiawww.conf`](https://github.com/decred/politeia/blob/master/politeiawww/sample-politeiawww.conf) and [`sample-politeiad.conf`](https://github.com/decred/politeia/blob/master/politeiad/sample-politeiad.conf) files.